src/artemis/experiment_plans/rotation_scan_plan.py

```diff
@@ -282,7 +239,22 @@ def rotation_scan_plan_with_stage_and_cleanup(
 
         @bpp.stage_decorator([eiger])
         @bpp.finalize_decorator(lambda: cleanup_plan(**devices))
-        def rotation_with_cleanup_and_stage(params):
+        def rotation_with_cleanup_and_stage(params: RotationInternalParameters):
+            LOGGER.info("setting up sample environment...")
+            yield from setup_sample_environment(
+                devices["detector_motion"],
+                devices["backlight"],
+                devices["attenuator"],
+                params.artemis_params.ispyb_params.transmission_fraction,
+                params.artemis_params.detector_params.detector_distance,
+            )
+            LOGGER.info("moving to position (if specified)")
+            yield from move_x_y_z(
+                devices["smargon"],
+                params.experiment_params.x,
+                params.experiment_params.y,
+                params.experiment_params.z,
+            )
             LOGGER.info("setting up and staging eiger...")
```

Comment:
Should: This comment is in the wrong place now, we started staging the eiger way before this